notebook/analysis/simulation_results.ipynb

```diff
@@ -426,32 +837,32 @@
     {
      "data": {
       "text/plain": [
-       "<ggplot: (8733008535785)>"
+       "<ggplot: (8761973210301)>"
       ]
      },
-     "execution_count": 8,
+     "execution_count": 10,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "plot = ggplot(sepsis_metrics, aes(x='train_count', y='balanced_accuracy', color='supervised')) \n",
+    "plot = ggplot(tb_metrics, aes(x='train_count', y='balanced_accuracy', color='supervised')) \n",
```

Comment:
Why do you think the `three_layer_net` and `pytorch_lr` models require so many more samples than the other models? Do you think more parameter tuning/model tweaking would help with this?